### DISCUSSION TITLE
Failed to select a version for `zeroize`

### DISCUSSION THREAD

**Discussion by luongngocminh** [aptos-labs/aptos-developer-discussions#421] [Questions]
### Discord user ID

188339997382934528

### Describe your question in detail.

I'm working on integrating the Aptos Rust SDK but there seem to be a dependency bug with the crate 'zeroize' when compiling

![image](https://github.com/user-attachments/assets/0ce516ac-1abc-4fd7-8369-6a0ae51e1812)

Here are my Cargo.toml and config.toml
![image](https://github.com/user-attachments/assets/8367ee47-d989-42d7-9b90-852735d805b9)
![image](https://github.com/user-attachments/assets/8c343702-2f0e-4bbf-af57-3fb5f41599ec)

### What error, if any, are you getting?

```
error: failed to select a version for `zeroize`.
    ... required by package `x25519-dalek v1.2.0`
    ... which satisfies dependency `x25519-dalek = "^1.2.0"` of package `aptos-crypto v0.0.3 (https://github.com/aptos-labs/aptos-core?branch=devnet#70806ff5)`
    ... which satisfies git dependency `aptos-crypto` of package `aptos-sdk v0.0.3 (https://github.com/aptos-labs/aptos-core?branch=devnet#70806ff5)`
    ... which satisfies git dependency `aptos-sdk` of package `contract v0.1.0 (/Users/luongminh/workspace/decentralized-llm-inference/crates/contract)`
versions that meet the requirements `=1.3` are: 1.3.0

all possible versions conflict with previously selected packages.

  previously selected package `zeroize v1.8.1`
    ... which satisfies dependency `zeroize = "^1.1.1"` (locked to 1.8.1) of package `tiny-bip39 v0.8.2`
    ... which satisfies dependency `tiny-bip39 = "^0.8.2"` of package `aptos-sdk v0.0.3 (https://github.com/aptos-labs/aptos-core?branch=devnet#70806ff5)`
    ... which satisfies git dependency `aptos-sdk` of package `contract v0.1.0 (/Users/luongminh/workspace/decentralized-llm-inference/crates/contract)`

failed to select a version for `zeroize` which could resolve this conflict
```

### What have you tried or looked at?  Or how can we reproduce the error?

Just follow the Rust SDK Installation Doc: https://aptos.dev/en/build/sdks/rust-sdk

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

Rust SDK

**Comment by martinsander00** [comment]
where you able to solve this?
